src/TestSuite/IntegrationTestCase.php

```diff
@@ -882,14 +913,15 @@ public function assertResponseEquals($content, $message = '')
      *
      * @param string $content The content to check for.
      * @param string $message The failure message that will be appended to the generated message.
+     * @param bool   $ignoreCase A flag to check whether we should ignore case or not.
```

Comment:
@viraj-khatavkar I think so. We never align docblocks parameters as they can become quickly out of sync.